sdk/node/src/api/mockHsmKeys.js

```diff
@@ -5,7 +5,7 @@ const shared = require('../shared')
   * Cryptographic private keys are the primary authorization mechanism on a
   * blockchain. For development environments, Chain Core provides a convenient
   * MockHSM.
-  * <br/><br/>
+  * 
```

Comment:
Yup. In this case, changing https://github.com/chain/chain/pull/931/files#diff-84c1d078debc432e4577dbe04061249aR16 to support https://github.com/chain/chain/pull/931/files#diff-f138fcf4ae92f3121a6c5988001b5f6fR4 broke a bunch of formatting elsewhere. I can split it up easily enough if that's preferred.